scripts/tests/test_validate_exception_location.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from scripts.validate_exception_location import validate_exception_and_warning_placement
+
+PATH = "t.py"
+
+# ERRORS_IN_TESTING_RST is the set returned when parsing testing.rst for all the
+# exceptions and warnings.
+CUSTOM_EXCEPTION_NOT_IN_TESTING_RST = "MyException"
+CUSTOM_EXCEPTION__IN_TESTING_RST = "MyOldException"
+ERRORS_IN_TESTING_RST = {CUSTOM_EXCEPTION__IN_TESTING_RST}
+
+TEST_CODE = """
+import numpy as np
+import sys
+
+def my_func():
+  pass
+
+class {custom_name}({error_type}):
+  pass
+
+"""
+
+# Test with various python-defined exceptions to ensure they are all flagged.
+testdata = [
```

Comment:
Could make this a pytest fixture that is parameterized over these values instead.

Reply:
I think I changed this right. Thank you for the suggestion. I didn't know I could it that way